runelite-client/src/main/java/net/runelite/client/plugins/idlenotifier/IdleNotifierPlugin.java

```diff
@@ -357,7 +350,7 @@ private boolean checkIdleLogout()
 		{
 			notifyIdleLogout = true;
 		}
-
+		
```

Comment:
Remove this whitespace

Reply:
I don't know why it's doing the add/remove line thing, but it doesn't actually add any whitespace

Reply:
Well based on diff you clearly added whitespace there